linux_os/guide/services/ssh/ssh_server/sshd_disable_x11_forwarding/oval/shared.xml

```diff
@@ -0,0 +1 @@
+{{{ oval_sshd_config(parameter="X11Forwarding", value="no", missing_parameter_pass=true) }}}
```

Comment:
@eradot4027 the oval file shouldn't need to be created as the template in rule.yml should create it for you.

Reply:
Removed